linux_sql/psql_docker.sh

```diff
@@ -0,0 +1,55 @@
+#! /bin/bash
+
+#make sure the correct number of arguments are given
+if [ $# -lt 1 ] || [ $# -gt 2 ];
+then
+	echo "Wrong number of arugments"
+	exit 1
+fi
+
+# Check that the command is to start or stop docker
+if [ $1 != "start" ] || [ $1 != "stop" ]
```

Comment:
you must assign args to variables. Please review coding style https://www.notion.so/jarviscanada/Shell-Style-Guide-764a3ac0c9044fdb844b55420d1c7594